lib/hmmlearn/base.py

```diff
@@ -544,36 +544,27 @@ def _fit_log(self, X):
         return log_frameprob, log_prob, posteriors, fwdlattice, bwdlattice
 
     def _do_viterbi_pass(self, log_frameprob):
-        state_sequence, log_prob = _hmmc.viterbi(
-            log_mask_zero(self.startprob_), log_mask_zero(self.transmat_),
+        return _hmmc.viterbi(self.startprob_, self.transmat_,
```

Comment:
I would even just get rid of these `_do_foo_pass` wrappers and directly inline the calls to the C hmmc functions at the call sites.

Reply:
Done.